src/Core/Collision/PolygonalCollider.cpp

```diff
@@ -208,9 +280,42 @@ namespace obe::Collision
     {
         CollisionData collData;
         collData.offset = offset;
+
+        //AABB filtering
```

Comment:
Maybe put this code inside a private function (not a method) so it doesn't appear twice ?